messaging/src/main/java/org/axonframework/common/BuilderUtils.java

```diff
@@ -54,11 +54,34 @@ private BuilderUtils() {
      * containing the provided {@code exceptionMessage}.
      *
      * @param value a {@code T} specifying the value to assert
+     * @param exceptionMessage The message for the exception.
```

Comment:
Super nitty, but throughout the javadoc I believe we try to follow the standard that `@param` and `@return` lines do not start with a capital or end with punctuation.
That would thus mean this line would be rewritten to: `* @param exceptionMessage the message for the exception`

Reply:
Similar comments hold for the newly introduced `assertPositive(int, String)` and `assertStrictPositive(int, String)`